### PR TITLE
Latest changes for introduction tutorial

### PR DESCRIPTION
**Pull request by devtastic**
## Description
Blog content for first tutorial with an aim to replace the current core blog for PyGrid beginners.

## Affected Dependencies
None

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [X] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [X] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [X] My changes are covered by tests

### CODE REVIEW

File: examples/data-centric/introduction/01-introduction-to-pygrid.ipynb [hunk 1]
```diff
@@ -0,0 +1,741 @@
...
+    "\n",
+    "Once done, you should have the following components:\n",
+    "\n",
+    " - PyGrid Network      (http://localhost:5000)\n",
+    " - PyGrid Node Bob (http://localhost:3000)\n",
+    " - PyGrid Node Alice   (http://localhost:3001)\n",
+    " \n",
+    "Yours URL / ports might vary depending on your configuration, so please ensure you look at the output of `docker-compose up` before moving forward.\n",
+    "\n",
+    "Visit http://localhost:5000/connected-nodes to ensure all nodes are ready and setup! You should get a response like below for the standard setup:\n",
```
Comment (nahuakang) [inline examples/data-centric/introduction/01-introduction-to-pygrid.ipynb:183]:
```suggestion
    "Visit http://localhost:7000/connected-nodes to ensure all nodes are ready and setup! You should get a response like below for the standard setup:\n",
```

File: examples/data-centric/introduction/01-introduction-to-pygrid.ipynb [hunk 2]
```diff
@@ -0,0 +1,741 @@
...
+    "hook = sy.TorchHook(th)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## __Step 2: Connect your node__\n",
+    "\n",
+    "The next step is to connect with your own node. It’s important to note that the node app was deployed in some environment and you need to know its address previously. In this case we’ll connect with the hospital node (Bob) on our `http://localhost:3000`."
```
Comment (nahuakang) [inline examples/data-centric/introduction/01-introduction-to-pygrid.ipynb:208]:
```suggestion
    "The next step is to connect with your own node. It’s important to note that the node app was deployed in some environment and you need to know its address previously. In this case we’ll connect with the hospital node (Bob) on our `http://localhost:5001`."
```

File: examples/data-centric/introduction/01-introduction-to-pygrid.ipynb [hunk 3]
```diff
@@ -0,0 +1,741 @@
...
+   "execution_count": 28,
+   "metadata": {
+    "tags": [
+     "parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "grid_address = \"http://localhost:5000\"\n",
+    "bob = \"http://localhost:3000\""
```
Comment (nahuakang) [inline examples/data-centric/introduction/01-introduction-to-pygrid.ipynb:222]:
```suggestion
    "grid_address = \"http://network:7000\"\n",
    "bob = \"http://bob:5001\""
```

File: examples/data-centric/introduction/01-introduction-to-pygrid.ipynb [hunk 4]
```diff
@@ -0,0 +1,741 @@
...
+    "### __Step 1: Import PySyft and dependencies__\n",
+    "To publish the sensitive yet valuable data, the data owner first imports PyGrid and PySyft dependencies.\n",
+    "\n",
+    "In this example, we’ll import `syft` and wrap the standard `torch` modules with `syft.TorchHook` to add the additional privacy-preserving features that PySyft offers. If you have not set up your computer, follow [this](https://github.com/OpenMined/PyGrid) to install the necessary dependencies.\n",
+    "\n",
+    "Also, please make sure [these](https://github.com/OpenMined/PyGrid/tree/dev/examples#running-the-examples) instructions have been followed.\n",
+    "\n",
+    "Once done, you should have the following components:\n",
+    "\n",
+    " - PyGrid Network      (http://localhost:5000)\n",
```
Comment (nahuakang) [inline examples/data-centric/introduction/01-introduction-to-pygrid.ipynb:177]:
```suggestion
    " - PyGrid Network      (http://network:7000)\n",
```

File: examples/data-centric/introduction/01-introduction-to-pygrid.ipynb [hunk 5]
```diff
@@ -0,0 +1,741 @@
...
+    " - PyGrid Node Bob (http://localhost:3000)\n",
```
Comment (nahuakang) [inline examples/data-centric/introduction/01-introduction-to-pygrid.ipynb:178]:
```suggestion
    " - PyGrid Node Bob (http://bob:5001)\n",
```

File: examples/data-centric/introduction/01-introduction-to-pygrid.ipynb [hunk 6]
```diff
@@ -0,0 +1,741 @@
...
+    " - PyGrid Node Alice   (http://localhost:3001)\n",
```
Comment (nahuakang) [inline examples/data-centric/introduction/01-introduction-to-pygrid.ipynb:179]:
```suggestion
    " - PyGrid Node Alice   (http://alice:5000)\n",
```

File: examples/data-centric/introduction/01-introduction-to-pygrid.ipynb (hunk 1 above)
Comment (nahuakang) [inline examples/data-centric/introduction/01-introduction-to-pygrid.ipynb:183]:
```suggestion
    "Visit http://network:7000/connected-nodes to ensure all nodes are ready and setup! You should get a response like below for the standard setup:\n",
```

File: examples/data-centric/introduction/01-introduction-to-pygrid.ipynb [hunk 7]
```diff
@@ -0,0 +1,741 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "feb_records.get()\n",
+    "# GETPERMITTEDERROR"
```
Comment (nahuakang) [inline examples/data-centric/introduction/01-introduction-to-pygrid.ipynb:536]:
```suggestion
    "# GetNotPermittedError"
```

**Review by nahuakang — COMMENTED** [review]
~Shall we remove the `examples/Beginner` directory then?~

Added some commit suggestions to the port numbers now :smile:

File: examples/data-centric/introduction/01-introduction-to-pygrid.ipynb (hunk 1 above)
Comment (jmaunon) [reply examples/data-centric/introduction/01-introduction-to-pygrid.ipynb:183]: I think that if you visit that address from your web browser, must be localhost.
The main rule is
* If the containers must talk to each other use custom hostname, i.e. "alice"
* If you want to visit something from your laptop, then, use localhost.

In any case, I will review this as well

Comment (nahuakang) [reply examples/data-centric/introduction/01-introduction-to-pygrid.ipynb:183]: I tried not using `localhost` and it worked in my Chrome. But sure that's a safe rule of thumbs!

**Review by jmaunon — APPROVED** [review]

**Review by nahuakang — APPROVED** [review]
Tried and all worked. Thanks a lot @devtastic  @jmaunon :rocket:

**Review by cereallarceny — APPROVED** [review]

**Pull request merged**